### PR TITLE
Graphics/animal jump animation

### PR DESCRIPTION
**Pull request by JobEngel**

### CODE REVIEW

**Comment by newnottakenname** [comment]
LGTM

File: desktop/src/main/nl/tudelft/ti2806/riverrush/graphics/entity/AnimalActor.java [hunk 1]
```diff
@@ -178,32 +234,61 @@ public void updateFlag(final Direction direction) {
...
+        // wiggleRight.setDuration(WIGGLE_RIGHT_DURATION);
+        //
+        // RotateByAction wiggleBack = Actions.rotateBy(WIGGLE_DISTANCE);
+        // wiggleBack.setDuration(WIGGLE_BACK_DURATION);
+        //
+        // SequenceAction wiggle = sequence(wiggleLeft, wiggleRight, wiggleBack);
+        //
+        // SequenceAction jump = sequence(jumpUp,
+        // Actions.repeat((int) (DELAY_DURATION / WIGGLE_DURATION), wiggle), drop);
+
```
Comment (txsmith) [inline desktop/src/main/nl/tudelft/ti2806/riverrush/graphics/entity/AnimalActor.java]: WIll merge when these comments are gone.

**Comment by txsmith** [comment]
Please also remove the images without transparancy

**Comment by JobEngel** [comment]
I had to merge this myself because git had some issues.

**Pull request merged**